InteractionController.cs

```diff
@@ -70,6 +86,66 @@ void PrintBooks(List<Book> books)
             .ForEach(Console.WriteLine);
     }
 
+    // from BookHelper
+    static void PrintUpdatedBookProperties(string updatedBook)
```

Comment:
Let's remove the static functions for now

Reply:
Removed static functions